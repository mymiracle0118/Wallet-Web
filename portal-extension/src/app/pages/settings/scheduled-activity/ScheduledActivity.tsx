import React from 'react'
import TransactionActivity from 'pages/wallet/token/transaction/activity/TransactionActivity'

type ScheduledActivityProps = {
  swapTokens?: boolean
}

const ScheduledActivity = ({ swapTokens = false }: ScheduledActivityProps) => {
  return <TransactionActivity swapTokens={swapTokens} txStatus="Completed" page="Settings" network="mainnet" />
}

export default ScheduledActivity
