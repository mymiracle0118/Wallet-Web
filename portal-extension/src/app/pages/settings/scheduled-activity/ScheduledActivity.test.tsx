import React from 'react'
import { render } from '@testing-library/react'
import { QueryClient, QueryClientProvider } from '@tanstack/react-query'

import ScheduledActivity from './ScheduledActivity'

describe('Send Review UI', () => {
  const queryClient = new QueryClient()

  test('renders correctly for Single Token Activity', () => {
    const tree = render(
      <QueryClientProvider client={queryClient}>
        <ScheduledActivity swapTokens={false} />
      </QueryClientProvider>
    )
    expect(tree).toMatchSnapshot()
  })
  test('renders correctly for Swap Activity', () => {
    const tree = render(
      <QueryClientProvider client={queryClient}>
        <ScheduledActivity swapTokens />
      </QueryClientProvider>
    )
    expect(tree).toMatchSnapshot()
  })
})
