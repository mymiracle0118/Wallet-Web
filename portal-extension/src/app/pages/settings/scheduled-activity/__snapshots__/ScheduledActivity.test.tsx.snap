// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Send Review UI renders correctly for Single Token Activity 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="dark:bg-surface-dark bg-custom-white flex flex-col h-[600px] dark:text-custom-white text-custom-black"
      >
        <div
          class="flex flex-row relative justify-between items-center h-16"
        >
          <button
            class="h-[24px] w-[24px] p-1 ml-1 hover:bg-custom-grey10 dark:hover:bg-custom-white10 rounded-full flex items-center justify-center absolute"
            type="button"
          >
            <arrow-left.svg
              classname="text-xl stroke-custom-black dark:stroke-custom-white"
            />
          </button>
          <div
            class="flex-1 text-center"
          >
            <div
              class="text-custom-black dark:text-custom-white text-lg font-extrabold"
            >
              Activity
            </div>
          </div>
        </div>
        <div
          class="w-full h-full overflow-auto"
        >
          <div
            class="MuiBox-root css-1g4yje1"
          >
            <div
              class="MuiBox-root css-2lipdz"
            >
              <div
                class="MuiBox-root css-v18s91"
              >
                <div>
                  <svg
                    class="inline-block select-none shrink-0 fill-custom-black dark:fill-custom-white text-2xl"
                    style="height: 1em; width: 1em;"
                  >
                    <arrow-up-right.svg />
                  </svg>
                </div>
                <div
                  style="margin-left: 28px; flex: 1;"
                >
                  <div
                    class="text-h4 text-custom-black dark:text-custom-white"
                  >
                    Send
                  </div>
                  <div
                    class="text-subtitle1 text-custom-grey100 dark:text-custom-white40"
                  >
                    Invalid date
                  </div>
                </div>
                <div
                  style="text-align: right;"
                >
                  <div
                    class="text-h4 text-custom-black dark:text-custom-white"
                  >
                    0
                  </div>
                  <div
                    class="text-subtitle1 text-custom-black dark:text-custom-white"
                  >
                    Completed
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root css-19midj6"
            >
              <button
                class="flex items-center justify-center rounded-md hover:cursor-pointer font-extrabold text-sm w-full h-[44px] border-[1px] border-solid text-primary dark:text-custom-white border-primary dark:border-custom-white40 dark:hover:border-custom-white hover:border-primary-alt hover:text-primary-alt"
                type="button"
              >
                Token.viewTransactionExplorer
              </button>
            </div>
            <div
              class="MuiBox-root css-1rr4qq7"
            >
              <div
                class="MuiBox-root css-i0w5pg"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  Labels.from
                </div>
                <div
                  class="MuiBox-root css-192qrng"
                >
                  <div
                    class="text-custom-white border-[1px] border-solid h-8  border-custom-white40 rounded-xl flex items-center justify-center w-fit px-3"
                  >
                    <div
                      class="text-note text-inherit"
                    >
                      ...
                    </div>
                    <div
                      class="relative"
                    >
                      <button
                        aria-expanded="false"
                        class="ml-1 p-1 focus:outline-none"
                        id="headlessui-popover-button-:r0:"
                        type="button"
                      >
                        <copy.svg
                          class="fill-custom-white"
                        />
                      </button>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiBox-root css-i0w5pg"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  Labels.to
                </div>
                <div
                  class="MuiBox-root css-192qrng"
                >
                  <div
                    class="text-custom-white border-[1px] border-solid h-8  border-custom-white40 rounded-xl flex items-center justify-center w-fit px-3"
                  >
                    <div
                      class="text-note text-inherit"
                    >
                      ...
                    </div>
                    <div
                      class="relative"
                    >
                      <button
                        aria-expanded="false"
                        class="ml-1 p-1 focus:outline-none"
                        id="headlessui-popover-button-:r3:"
                        type="button"
                      >
                        <copy.svg
                          class="fill-custom-white"
                        />
                      </button>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiBox-root css-i0w5pg"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  Network.networkFee
                </div>
                <div
                  style="text-align: right;"
                >
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                  >
                    0
                     ETH
                  </div>
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-regular"
                  >
                    $0
                  </div>
                </div>
              </div>
              <div
                class="MuiBox-root css-19midj6"
              >
                <div
                  class="MuiBox-root css-7uyrno"
                >
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                  >
                    ScheduledTransaction.feeSavedByScheduling
                  </div>
                  <div
                    class="MuiBox-root css-gai91n"
                  >
                    <div
                      class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                    >
                      0.0014 WETH
                    </div>
                    <div
                      class="text-custom-black dark:text-custom-white text-sm font-regular"
                    >
                      $5.2514
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiBox-root css-0"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="dark:bg-surface-dark bg-custom-white flex flex-col h-[600px] dark:text-custom-white text-custom-black"
    >
      <div
        class="flex flex-row relative justify-between items-center h-16"
      >
        <button
          class="h-[24px] w-[24px] p-1 ml-1 hover:bg-custom-grey10 dark:hover:bg-custom-white10 rounded-full flex items-center justify-center absolute"
          type="button"
        >
          <arrow-left.svg
            classname="text-xl stroke-custom-black dark:stroke-custom-white"
          />
        </button>
        <div
          class="flex-1 text-center"
        >
          <div
            class="text-custom-black dark:text-custom-white text-lg font-extrabold"
          >
            Activity
          </div>
        </div>
      </div>
      <div
        class="w-full h-full overflow-auto"
      >
        <div
          class="MuiBox-root css-1g4yje1"
        >
          <div
            class="MuiBox-root css-2lipdz"
          >
            <div
              class="MuiBox-root css-v18s91"
            >
              <div>
                <svg
                  class="inline-block select-none shrink-0 fill-custom-black dark:fill-custom-white text-2xl"
                  style="height: 1em; width: 1em;"
                >
                  <arrow-up-right.svg />
                </svg>
              </div>
              <div
                style="margin-left: 28px; flex: 1;"
              >
                <div
                  class="text-h4 text-custom-black dark:text-custom-white"
                >
                  Send
                </div>
                <div
                  class="text-subtitle1 text-custom-grey100 dark:text-custom-white40"
                >
                  Invalid date
                </div>
              </div>
              <div
                style="text-align: right;"
              >
                <div
                  class="text-h4 text-custom-black dark:text-custom-white"
                >
                  0
                </div>
                <div
                  class="text-subtitle1 text-custom-black dark:text-custom-white"
                >
                  Completed
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiBox-root css-19midj6"
          >
            <button
              class="flex items-center justify-center rounded-md hover:cursor-pointer font-extrabold text-sm w-full h-[44px] border-[1px] border-solid text-primary dark:text-custom-white border-primary dark:border-custom-white40 dark:hover:border-custom-white hover:border-primary-alt hover:text-primary-alt"
              type="button"
            >
              Token.viewTransactionExplorer
            </button>
          </div>
          <div
            class="MuiBox-root css-1rr4qq7"
          >
            <div
              class="MuiBox-root css-i0w5pg"
            >
              <div
                class="text-custom-black dark:text-custom-white text-sm font-extrabold"
              >
                Labels.from
              </div>
              <div
                class="MuiBox-root css-192qrng"
              >
                <div
                  class="text-custom-white border-[1px] border-solid h-8  border-custom-white40 rounded-xl flex items-center justify-center w-fit px-3"
                >
                  <div
                    class="text-note text-inherit"
                  >
                    ...
                  </div>
                  <div
                    class="relative"
                  >
                    <button
                      aria-expanded="false"
                      class="ml-1 p-1 focus:outline-none"
                      id="headlessui-popover-button-:r0:"
                      type="button"
                    >
                      <copy.svg
                        class="fill-custom-white"
                      />
                    </button>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root css-i0w5pg"
            >
              <div
                class="text-custom-black dark:text-custom-white text-sm font-extrabold"
              >
                Labels.to
              </div>
              <div
                class="MuiBox-root css-192qrng"
              >
                <div
                  class="text-custom-white border-[1px] border-solid h-8  border-custom-white40 rounded-xl flex items-center justify-center w-fit px-3"
                >
                  <div
                    class="text-note text-inherit"
                  >
                    ...
                  </div>
                  <div
                    class="relative"
                  >
                    <button
                      aria-expanded="false"
                      class="ml-1 p-1 focus:outline-none"
                      id="headlessui-popover-button-:r3:"
                      type="button"
                    >
                      <copy.svg
                        class="fill-custom-white"
                      />
                    </button>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root css-i0w5pg"
            >
              <div
                class="text-custom-black dark:text-custom-white text-sm font-extrabold"
              >
                Network.networkFee
              </div>
              <div
                style="text-align: right;"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  0
                   ETH
                </div>
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-regular"
                >
                  $0
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root css-19midj6"
            >
              <div
                class="MuiBox-root css-7uyrno"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  ScheduledTransaction.feeSavedByScheduling
                </div>
                <div
                  class="MuiBox-root css-gai91n"
                >
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                  >
                    0.0014 WETH
                  </div>
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-regular"
                  >
                    $5.2514
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiBox-root css-0"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Send Review UI renders correctly for Swap Activity 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="dark:bg-surface-dark bg-custom-white flex flex-col h-[600px] dark:text-custom-white text-custom-black"
      >
        <div
          class="flex flex-row relative justify-between items-center h-16"
        >
          <button
            class="h-[24px] w-[24px] p-1 ml-1 hover:bg-custom-grey10 dark:hover:bg-custom-white10 rounded-full flex items-center justify-center absolute"
            type="button"
          >
            <arrow-left.svg
              classname="text-xl stroke-custom-black dark:stroke-custom-white"
            />
          </button>
          <div
            class="flex-1 text-center"
          >
            <div
              class="text-custom-black dark:text-custom-white text-lg font-extrabold"
            >
              Activity
            </div>
          </div>
        </div>
        <div
          class="w-full h-full overflow-auto"
        >
          <div
            class="MuiBox-root css-1g4yje1"
          >
            <div
              class="MuiBox-root css-2lipdz"
            >
              <button
                class="flex w-full height-[56px] items-center pt-2 pb-2 pl-4 pr-4 hover:cursor-pointer hover:bg-custom-grey10 dark:hover:bg-custom-white10 box-border"
                type="button"
              >
                <div>
                  <svg
                    class="inline-block select-none shrink-0 fill-custom-black dark:fill-custom-white text-4xl"
                    style="height: 1em; width: 1em;"
                  >
                    <dai.svg />
                  </svg>
                  <svg
                    class="inline-block select-none shrink-0 -ml-2 fill-custom-black dark:fill-custom-white text-4xl"
                    style="height: 1em; width: 1em;"
                  >
                    <weth.svg />
                  </svg>
                </div>
                <div
                  class="ml-4 flex-1 text-left"
                >
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                  >
                    DAI to WETH
                  </div>
                  <div
                    class="text-custom-grey100 dark:text-custom-white40 text-sm font-normal"
                  >
                    Invalid date
                  </div>
                </div>
                <div
                  class="text-right"
                >
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                  >
                    ≤ 0.04 WETH
                  </div>
                  <div
                    class="text-primary-alt dark:text-secondary text-sm font-normal"
                  >
                    Scheduled
                  </div>
                </div>
              </button>
            </div>
            <div
              class="MuiBox-root css-19midj6"
            >
              <button
                class="flex items-center justify-center rounded-md hover:cursor-pointer font-extrabold text-sm w-full h-[44px] border-[1px] border-solid text-primary dark:text-custom-white border-primary dark:border-custom-white40 dark:hover:border-custom-white hover:border-primary-alt hover:text-primary-alt"
                type="button"
              >
                Token.viewTransactionExplorer
              </button>
            </div>
            <div
              class="MuiBox-root css-1rr4qq7"
            >
              <div
                class="MuiBox-root css-i0w5pg"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  Labels.from
                </div>
                <div
                  class="MuiBox-root css-192qrng"
                >
                  <div
                    class="text-custom-white border-[1px] border-solid h-8  border-custom-white40 rounded-xl flex items-center justify-center w-fit px-3"
                  >
                    <div
                      class="text-note text-inherit"
                    >
                      ...
                    </div>
                    <div
                      class="relative"
                    >
                      <button
                        aria-expanded="false"
                        class="ml-1 p-1 focus:outline-none"
                        id="headlessui-popover-button-:r6:"
                        type="button"
                      >
                        <copy.svg
                          class="fill-custom-white"
                        />
                      </button>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiBox-root css-i0w5pg"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  Labels.to
                </div>
                <div
                  class="MuiBox-root css-192qrng"
                >
                  <div
                    class="text-custom-white border-[1px] border-solid h-8  border-custom-white40 rounded-xl flex items-center justify-center w-fit px-3"
                  >
                    <div
                      class="text-note text-inherit"
                    >
                      ...
                    </div>
                    <div
                      class="relative"
                    >
                      <button
                        aria-expanded="false"
                        class="ml-1 p-1 focus:outline-none"
                        id="headlessui-popover-button-:r9:"
                        type="button"
                      >
                        <copy.svg
                          class="fill-custom-white"
                        />
                      </button>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiBox-root css-i0w5pg"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  Labels.rate
                </div>
                <div
                  style="text-align: right;"
                >
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                  >
                    1 DAI ≥ 0.0004 WETH
                  </div>
                </div>
              </div>
              <div
                class="MuiBox-root css-i0w5pg"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  Token.gasFee
                </div>
                <div
                  style="text-align: right;"
                >
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                  >
                    Token.reatTimePrice
                  </div>
                </div>
              </div>
              <div
                class="MuiBox-root css-i0w5pg"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  Token.dueTime
                </div>
                <div
                  style="text-align: right;"
                >
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                  >
                    Oct. 5 at 9:36 PM
                  </div>
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-regular"
                  >
                    6 hours from now
                  </div>
                </div>
              </div>
              <div
                class="MuiBox-root css-19midj6"
              >
                <div
                  class="MuiBox-root css-7uyrno"
                >
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                  >
                    ScheduledTransaction.totalSavedByScheduling
                  </div>
                  <div
                    class="MuiBox-root css-gai91n"
                  >
                    <div
                      class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                    >
                      0.0014 WETH
                    </div>
                    <div
                      class="text-custom-black dark:text-custom-white text-sm font-regular"
                    >
                      $5.2514
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiBox-root css-0"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="dark:bg-surface-dark bg-custom-white flex flex-col h-[600px] dark:text-custom-white text-custom-black"
    >
      <div
        class="flex flex-row relative justify-between items-center h-16"
      >
        <button
          class="h-[24px] w-[24px] p-1 ml-1 hover:bg-custom-grey10 dark:hover:bg-custom-white10 rounded-full flex items-center justify-center absolute"
          type="button"
        >
          <arrow-left.svg
            classname="text-xl stroke-custom-black dark:stroke-custom-white"
          />
        </button>
        <div
          class="flex-1 text-center"
        >
          <div
            class="text-custom-black dark:text-custom-white text-lg font-extrabold"
          >
            Activity
          </div>
        </div>
      </div>
      <div
        class="w-full h-full overflow-auto"
      >
        <div
          class="MuiBox-root css-1g4yje1"
        >
          <div
            class="MuiBox-root css-2lipdz"
          >
            <button
              class="flex w-full height-[56px] items-center pt-2 pb-2 pl-4 pr-4 hover:cursor-pointer hover:bg-custom-grey10 dark:hover:bg-custom-white10 box-border"
              type="button"
            >
              <div>
                <svg
                  class="inline-block select-none shrink-0 fill-custom-black dark:fill-custom-white text-4xl"
                  style="height: 1em; width: 1em;"
                >
                  <dai.svg />
                </svg>
                <svg
                  class="inline-block select-none shrink-0 -ml-2 fill-custom-black dark:fill-custom-white text-4xl"
                  style="height: 1em; width: 1em;"
                >
                  <weth.svg />
                </svg>
              </div>
              <div
                class="ml-4 flex-1 text-left"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  DAI to WETH
                </div>
                <div
                  class="text-custom-grey100 dark:text-custom-white40 text-sm font-normal"
                >
                  Invalid date
                </div>
              </div>
              <div
                class="text-right"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  ≤ 0.04 WETH
                </div>
                <div
                  class="text-primary-alt dark:text-secondary text-sm font-normal"
                >
                  Scheduled
                </div>
              </div>
            </button>
          </div>
          <div
            class="MuiBox-root css-19midj6"
          >
            <button
              class="flex items-center justify-center rounded-md hover:cursor-pointer font-extrabold text-sm w-full h-[44px] border-[1px] border-solid text-primary dark:text-custom-white border-primary dark:border-custom-white40 dark:hover:border-custom-white hover:border-primary-alt hover:text-primary-alt"
              type="button"
            >
              Token.viewTransactionExplorer
            </button>
          </div>
          <div
            class="MuiBox-root css-1rr4qq7"
          >
            <div
              class="MuiBox-root css-i0w5pg"
            >
              <div
                class="text-custom-black dark:text-custom-white text-sm font-extrabold"
              >
                Labels.from
              </div>
              <div
                class="MuiBox-root css-192qrng"
              >
                <div
                  class="text-custom-white border-[1px] border-solid h-8  border-custom-white40 rounded-xl flex items-center justify-center w-fit px-3"
                >
                  <div
                    class="text-note text-inherit"
                  >
                    ...
                  </div>
                  <div
                    class="relative"
                  >
                    <button
                      aria-expanded="false"
                      class="ml-1 p-1 focus:outline-none"
                      id="headlessui-popover-button-:r6:"
                      type="button"
                    >
                      <copy.svg
                        class="fill-custom-white"
                      />
                    </button>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root css-i0w5pg"
            >
              <div
                class="text-custom-black dark:text-custom-white text-sm font-extrabold"
              >
                Labels.to
              </div>
              <div
                class="MuiBox-root css-192qrng"
              >
                <div
                  class="text-custom-white border-[1px] border-solid h-8  border-custom-white40 rounded-xl flex items-center justify-center w-fit px-3"
                >
                  <div
                    class="text-note text-inherit"
                  >
                    ...
                  </div>
                  <div
                    class="relative"
                  >
                    <button
                      aria-expanded="false"
                      class="ml-1 p-1 focus:outline-none"
                      id="headlessui-popover-button-:r9:"
                      type="button"
                    >
                      <copy.svg
                        class="fill-custom-white"
                      />
                    </button>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root css-i0w5pg"
            >
              <div
                class="text-custom-black dark:text-custom-white text-sm font-extrabold"
              >
                Labels.rate
              </div>
              <div
                style="text-align: right;"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  1 DAI ≥ 0.0004 WETH
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root css-i0w5pg"
            >
              <div
                class="text-custom-black dark:text-custom-white text-sm font-extrabold"
              >
                Token.gasFee
              </div>
              <div
                style="text-align: right;"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  Token.reatTimePrice
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root css-i0w5pg"
            >
              <div
                class="text-custom-black dark:text-custom-white text-sm font-extrabold"
              >
                Token.dueTime
              </div>
              <div
                style="text-align: right;"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  Oct. 5 at 9:36 PM
                </div>
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-regular"
                >
                  6 hours from now
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root css-19midj6"
            >
              <div
                class="MuiBox-root css-7uyrno"
              >
                <div
                  class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                >
                  ScheduledTransaction.totalSavedByScheduling
                </div>
                <div
                  class="MuiBox-root css-gai91n"
                >
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-extrabold"
                  >
                    0.0014 WETH
                  </div>
                  <div
                    class="text-custom-black dark:text-custom-white text-sm font-regular"
                  >
                    $5.2514
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiBox-root css-0"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
